articles/quickstart/spa/angular2/06-token-renewal.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Silent Token Renewal
+description: This tutorial demonstrates how to add automatic access_token renewal to an Angular 2+ application with Auth0
+budicon: 448
+---
+
+<%= include('../../../_includes/_package', {
+  org: 'auth0-samples',
+  repo: 'auth0-angular-samples',
+  path: '06-Token-Renewal',
+  requirements: [
+    'Angular 2+'
+  ]
+}) %>
+
+## Server Setup
+
+Using access_token renewal will require a server component. A simplistic one could be done like this:
```

Comment:
Reword this to:

"Renewing the user's `access_token` requires that a static HTML file to be served. Create a simple server with **express** and add a file called `silent.html`.